CHANGES.md

```diff
@@ -8,6 +8,8 @@ To be released.
 
 ### Deprecated APIs
 
+ -  Removed `PolymorphicAction<T>` class.  [[#3193]]
```

Comment:
IMO this is `Backward-incompatible API changes`.

Reply:
Done